modules/microshift-firewall-known-issue.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-firewall-known-issue_{context}"]
+== Known firewall issue
+To avoid breaking traffic flows with a firewall restart, execute firewall commands before starting {product-title}. The CSI driver in {product-title} makes use of iptable rules for some traffic flows, such as those using the NodePort service. The iptable rules are generated and inserted by the CSI driver, but are deleted when the firewall restarts. The absence of the iptable rules breaks traffic flows. If firewall commands have to be executed after {product-title} is running, manually restart {product-title} to reset the rules controlled by the CSI driver.
```

Comment:
```suggestion
To avoid breaking traffic flows with a firewall reload or restart, execute firewall commands before starting {product-title}. The CNI driver in {product-title} makes use of iptable rules for some traffic flows, such as those using the NodePort service. The iptable rules are generated and inserted by the CNI driver, but are deleted when the firewall reload or restart. The absence of the iptable rules breaks traffic flows. If firewall commands have to be executed after {product-title} is running, manually restart ovnkube-master pod in openshift-ovn-kubernetes namespace to reset the rules controlled by the CNI driver.
```

Reply:
@zshi-redhat Does the admin see the ovnkube-master pod in openshift-ovn-kubernetes namespace, or can they just restart MicroShift? Which is simpler to do?

Reply:
@ShaunaDiaz  Yes, admin can see the ovnkube-master pod by listing the pods in openshift-ovn-kubernetes namespace, for example:
```
oc get pods -n openshift-ovn-kubernetes
```
Restarting microshift may not always restart ovnkube-master pod, so there is chance that restarting microshift won't fix the iptable issue.